base/ltcounts.dtx

```diff
@@ -507,9 +498,9 @@
     \@addtoreset{#3}{#4}%
     \IfBooleanF #1%
        {\expandafter
-         \gdef\csname the#3\expandafter\endcsname
-         \expandafter
-             {\csname the#4\endcsname .#2{#3}}}%
+         \xdef\csname the#3\expandafter\endcsname
+            {\expandafter\noexpand\csname the#4\endcsname
+             .\noexpand#2{#3}}}%
```

Comment:
Is `#2` forced to be a single token, usually a command which takes one mandatory argument? Do we need `\unexpanded{#2}` here?

Reply:
It is documented to be a single token (format-cmd) but I guess you are right, it doesn't hurt to use unexpanded in this place.